### ISSUE TITLE
Heatmap cell sizes can change while hovering across a row

### ISSUE THREAD

**Issue by infield-bookie** [highcharts/highcharts#17615]
#### Expected behaviour
Heatmap cell sizes remain consistent and clearly defined on hover

#### Actual behaviour
Heatmap cell sizes can change while hovering across a row and may vary depending on most recent hover state.

#### Live demo with steps to reproduce
https://jsfiddle.net/sxw48qt9/

#### Product version
10.2

**Comment by pawellysy** [comment]
Hi @infield-bookie, thanks for contacting us with your issue!
TL;DR: So everything works correctly. 
Unless you declare the [rowsize](https://api.highcharts.com/highcharts/series.heatmap.rowsize) or [columnsize](https://api.highcharts.com/highcharts/series.heatmap.columnsize) heatmap cell has a width and height of 1 in the units of the axis. This means, that if you define the data as you did, some points overlap each other. This means, that some points are beneath the other, and the index of the changes on hover so that the hovered point can be visible. 

In order to adjust the chart to your needs, please contact us via our support channel, where our support engineers will help you tweak your configuration to your needs. Thanks!
